kustomize/data-plane/prerequisites/service-mesh/resources/service-mesh-control-plane.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: maistra.io/v2
```

Comment:
@winklerm We seem to be duplicating resources here from the `kustomize/base-openshift/shard` directory. What's the reason we need to do that please?

Reply:
@winklerm Let's take it step-by-step, but it's important that we have the Jira task in our backlog to remove the duplicates so that we keep our setup clean and that the rest of the team isn't confused as to why there is so much duplication.

So I'd break it down into two/three steps, something like:

1. Create the prerequisites directory structure here with overlay for your environment only and verify that it works
2. Follow up PR to create overlay for `dev` environment, removing the pre-requisites stuff from the `base-openshift` directory with email to team on `kustomize` structure changes